src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -41,9 +41,7 @@ const useClickTrackerHandler = (eventTrackingData = {}) => {
   const { service, useReverb } = useContext(ServiceContext);
 
   const { optimizely } = useContext(OptimizelyContext);
-  const optimizelyVariation = useOptimizelyMvtVariation(
-    OPTIMIZELY_CONFIG.ruleKey,
-  );
+  const optimizelyVariation = useOptimizelyVariation({ flagKey: OPTIMIZELY_CONFIG.ruleKey });
```

Comment:
Can we just name this `experimentVariant` so that we don't have to do `experimentVariant: optimizelyVariation` below?